src/components/OcButton.spec.js

```diff
@@ -0,0 +1,53 @@
+import OcButton from "./OcButton.vue";
+import {createLocalVue, shallowMount} from "@vue/test-utils";
+const localVue = createLocalVue()
+
+
+describe("oc button", () => {
```

Comment:
Please use the actual component name here:
```suggestion
describe("OcButton", () => {
```